### ISSUE TITLE
Open Images V7 dataset downloader does not build labels properly

### ISSUE THREAD

**Issue by Yusepp** [ultralytics/ultralytics#4982]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Other

### Bug

- YOLOv8.0.181 
- OS: PopOS 22.04
- Python 3.10.12
- torch 2.1.0

### Environment

- YOLOv8.0.181 
- OS: PopOS 22.04
- Python 3.10.12
- torch 2.1.0

### Minimal Reproducible Example

```python
from ultralytics import YOLO

model = YOLO('yolov8m.pt', task='detect')

results = model.train(data='open-images-v7.yaml', epochs=10, imgsz=640)
# Default .yaml like the provided in the docs for this dataset
```

In train time i found the model only trains classification because there's no labels.

Also i checked the labels folder and all the  .txt are created but empty.

Any idea?

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@Yusepp OIv7 downloads and trains correctly. This is an extremely large dataset which requires over 1 TB (1000 GB) of storage space for download and unzip, and the download process itself may take several days.

**Comment by glenn-jocher** [comment]
Fore more information see https://docs.ultralytics.com/datasets/detect/open-images-v7/
